Pester.psm1

```diff
@@ -659,6 +659,48 @@ function Get-ScriptBlockScope
     [scriptblock].GetProperty('SessionStateInternal', $flags).GetValue($ScriptBlock, $null)
 }
 
+function Get-OperatingSystem
```

Comment:
This function uses magic strings, and we don't know yet that we will need to differentiate among multiple operating systems. At the moment this function is only used to distinguish Windows from non Windows systems so I suggest you make it `Test-Windows` function that will return Boolean (better name? IsWindows?).